src/main/kotlin/com/hibob/academy/employeeFeedback/Service/FeedbackCreator.kt

```diff
@@ -0,0 +1,17 @@
+package com.hibob.academy.employeeFeedback.Service
+
+import com.hibob.academy.employeeFeedback.dao.CreationFeedback
+import com.hibob.academy.employeeFeedback.dao.FeedbackDao
+import com.hibob.academy.employeeFeedback.dao.UserLoggedInDetails
+
+
+class FeedbackCreator(private val feedbackDao: FeedbackDao) {
+    fun createFeedBack(userLoggedInDetails: UserLoggedInDetails, feedback: CreationFeedback): Long {
```

Comment:
`FeedbackCreationRequest` is a better name, but you can rename it later in a different PR. I don't want it to stop you

Reply:
i will do it in another pr later .